.tekton/pull-request.yaml

```diff
@@ -124,7 +124,7 @@ spec:
               - name: APP_SUFFIX
                 value: "{{ pull_request_number }}"
               - name: COMPONENT_REPO_URLS
-                value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test"
+                value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test,https://github.com/redhat-appstudio-qe/fbc-sample-repo"
```

Comment:
please add network check to https://github.com/redhat-appstudio-qe/fbc-sample-repo - it should be always hermetic [STONEBLD-31](https://issues.redhat.com/browse/STONEBLD-31)